templates/shared/issuelist.tmpl

```diff
@@ -66,17 +66,17 @@
 					{{if .IsPull}}
 						<div class="branches df ac">
 							<div class="branch">
-								<a href="{{.PullRequest.BaseRepo.HTMLURL}}/src/branch/{{PathEscapeSegments .PullRequest.BaseBranch}}">
+								<a class="bold dif" href="{{.PullRequest.BaseRepo.HTMLURL}}/src/branch/{{PathEscapeSegments .PullRequest.BaseBranch}}">
 									{{/* inline to remove the spaces between spans */}}
-									{{if ne .RepoID .PullRequest.BaseRepoID}}<span class="truncated-name">{{.PullRequest.BaseRepo.OwnerName}}</span>:{{end}}<span class="truncated-name">{{.PullRequest.BaseBranch}}</span>
+									{{if ne .RepoID .PullRequest.BaseRepoID}}<span class="truncated-name dib vb">{{.PullRequest.BaseRepo.OwnerName}}</span>:{{end}}<span class="truncated-name dib vb">{{.PullRequest.BaseBranch}}</span>
```

Comment:
`vb` does not appear to be a [valid helper class](https://github.com/go-gitea/gitea/blob/main/web_src/less/helpers.less).

Reply:
deleted